CRM/Member/BAO/Membership.php

```diff
@@ -2272,7 +2272,7 @@ public static function updateAllMembershipStatus() {
       self::processOverriddenUntilDateMembership($dao1);
     }
 
-    $query = $baseQuery . " AND civicrm_membership.is_override IS NULL
+    $query = $baseQuery . " AND civicrm_membership.is_override != 1 
```

Comment:
This won't match `is_override IS NULL`.  You need:

`AND (civicrm_membership.is_override IS NULL OR civicrm_membership.is_override == 0)`

Reply:
such correct.  indeed i was backwards